Update harness-platform-faqs.md (#6495)
harness/developer-hub: kb/platform/harness-platform-faqs.md

### Can you link the docs to the API endpoint used to authorize a user with API token?

> Harness has an API to to check whether you perform Authorization using bearer token or API key token. For more information, go to [Get access control list](https://apidocs.harness.io/tag/Access-Control-List#operation/getAccessControlList) in the API documentation. For API token categories: Service Account vs. Personal access tokens, go to [Add and mange API keys](/docs/platform/automation/api/add-and-manage-api-keys/).

Reviewer: ```suggestion
Harness has an API to to check whether you perform authorization using bearer or API key token. For more information, go to [Get access control list](https://apidocs.harness.io/tag/Access-Control-List#operation/getAccessControlList) in the API documentation. For API token categories: Service Account vs. Personal access tokens, go to [Add and manage API keys](/docs/platform/automation/api/add-and-manage-api-keys/).
```

### How can we export users from Harness?

Currently, Harness doesn't support direct functionality for exporting all users. To export users from Harness, you can utilize the getUsers API. For more information, go to [getUsers](https://apidocs.harness.io/tag/User#operation/getUsers) in the API documentation.

### Does Harness AIDA support APIs for developers to create custom AI/ML solutions?

No. AIDA does not offer Rest APIs to be used by Harness users.

### How can I export all AutoStopping rules?

> Currently, Harness doesn't support direct functionality for exporting all AutoStopping rules. To create a report of all your AutoStopping rules, we recommend using the `List AutoStopping Rules` API method.

Reviewer: ```suggestion
Currently, Harness doesn't support direct functionality for exporting all AutoStopping rules. To create a report of all your AutoStopping rules, Harness recommends using the `List AutoStopping Rules` API method.
```

However, once 2FA is enabled by a user, only that specific user can disable it. Account administrators are unable to disable 2FA for individual users once it has been activated by the user themselves.

### Why am I prompted for 2FA when other users aren't?

You can manage 2FA in two ways:

**Individual user:** you can set up 2FA for your own User Profile without impacting other user accounts.
**All account users:** if you have Create/Edit permissions for Authentication Settings, you can enforce 2FA for all users in Harness. First, you set up 2FA for your own account, and then you can enforce 2FA account-wide in the Harness account's Login Settings.

To check whether 2FA is enabled for your user account, select your User Profile. You can toggle the **Two-Factor Authentication** indicator. For more information, go to [Two-factor authentication](/docs/platform/authentication/two-factor-authentication/).

> 
> the user profile to check and confirm. This can be enabled on the user level and can differ from other user settings.
> 

Reviewer: ```suggestion
```
